src/Compilers/CSharp/Test/Emit2/Semantics/ExtensionTypeTests.cs

```diff
@@ -21501,23 +21652,16 @@ static class C
         // PROTOTYPE should warn about hiding
         // PROTOTYPE update overload resolution so that extension members can hide members of the underlying type
```

Comment:
>// PROTOTYPE update overload resolution so that extension members can hide members of the underlying type [](http://example.com/codeflow?start=8&length=105)

Is this comment still relevant?